src/index.html

```diff
@@ -1,12 +1,389 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header page__header">
+      <a href="#" class="header__logo-link">
+        <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="MOYO logo">
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/134778857/952b2a4b-af42-4915-abe5-ee95ced4befb)
Change the button background to #00acdc on hover was on task requirements.
Problem with position was because of wrong media